src/Ztm.Zcoin.Rpc.Tests/ZcoinRpcClientTests.cs

```diff
@@ -558,15 +572,16 @@ public async Task GetPropertyBalanceAsync_WithValidArgs_ShouldSuccess()
         public async Task GetPropertyBalanceAsync_WithDivisibleToken_ShouldSuccess()
         {
             // Arrange.
-            var builder = new PropertyBuilder(this.node, this.subject);
-            builder.propertyType = PropertyType.Divisible;
-            var property = await builder.BuildAsync(CancellationToken.None);
+            var issuer = new PropertyIssuer(this.node, this.subject);
+            issuer.PropertyType = PropertyType.Divisible;
+            this.node.Generate(101);
+            var property = await issuer.IssueManagedAsync();
 
-            await property.GrantAsync(10_0000_0000);
+            await property.GrantAsync(new PropertyAmount(10_0000_0000));
```

Comment:
We should use `PropertyAmount.FromDivisible` instead so the logic to transform value will be in a single place.